src/spin.rs

```diff
@@ -0,0 +1,16 @@
+use std::path::PathBuf;
+use std::process::Command;
+
+pub fn plugin_install_file(file: PathBuf) -> Result<(), std::io::Error> {
+    let spin = std::env::var("SPIN_BIN_PATH").unwrap_or("spin".into());
+
+    Command::new(spin)
+        .arg("plugin")
+        .arg("install")
+        .arg("--file")
+        .arg(file)
```

Comment:
Might want `--yes` here, otherwise Spin will prompt, which seems unnecessary in this situation.

Reply:
Makes sense! Added